examples/quickstart_aws/quickstart_aws/assets/hackernews.py

```diff
@@ -78,11 +78,11 @@ def hackernews_topstories_word_cloud(
 
     # Also, upload the image to S3
     bucket_name = os.environ.get("S3_BUCKET")
-    bucket_location = context.resources.s3.get_bucket_location(Bucket=bucket_name)[
+    bucket_location = context.resources.s3.get_client().get_bucket_location(Bucket=bucket_name)[
```

Comment:
```suggestion
    bucket_location = s3.get_client().get_bucket_location(Bucket=bucket_name)[
```
if you change signature above